### DISCUSSION TITLE
Help please i want registre to github student developer pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35523] [GitHub Education]
Hello dear community I want registre to github global campus as student, my school is in list. But when i sent a photo of my reference from director i get an error. My raference was writed in Russian language

**Comment by fadyat** [comment]
Hello, brother 😃 

Check why your previous application was rejected.
As far as I remember, you must also add a translation of your document confirming that you are a student.
Check more information at https://education.github.com/discount_requests/pack_application

If you really are from a school that is on the list, you should be fine.
Just make the correct request again by analyzing the previous errors

GL

**Comment by dennewbie** [comment]
I think you could use your college/uni's email for verification and, furthermore, in case your director reference is not valid as a certificate, you could ask for a document at your "Student Administration Office" (or something like that) that certify you as a student in that college/uni, take a picture and hope that it's a valid method. It's really common that even if the college/uni is related to a foreign country, the "Student Administration Office" has the possibility to "render/compile/produce" it in a different language.
